autogen/agentchat/groupchat.py

```diff
@@ -61,6 +61,8 @@ def custom_speaker_selection_func(
         "clear history" phrase in user prompt. This is experimental feature.
         See description of GroupChatManager.clear_agents_history function for more info.
     - send_introductions: send a round of introductions at the start of the group chat, so agents know who they can speak to (default: False)
+    - custom_select_speaker_msg: customize the select speaker message. The string value will be converted to an f-string, use "{roles}" to output the agent's and their role descriptions and "{agentlist}" for a comma-separated list of agent names in square brackets.
```

Comment:
Moving this and the `custom_select_speaker_prompt` to above `speaker_selection_method` as they belong to the same group. 

Maybe also rename to `select_speaker_prompt_template` and `select_speaker_message_template` for consistency?

Reply:
Another thing would be useful is to show the default values here.

Reply:
Useful to comment on the different between the `select_speaker_message_template` and `select_speaker_prompt_tempalte`.

Reply:
All good points, I'll update